tests/e2e/features/test_e2e_add_new_grant_and_back_to_dashboard.py

```diff
@@ -0,0 +1,32 @@
+from playwright.sync_api import Page
+
+from tests.e2e.pages.create_grant_page import CreateGrantPage
+from tests.e2e.pages.dashboard_page import DashboardPage
+from tests.e2e.pages.grants_page import GrantsPage
+
+
+# TC-002-[1]: Add new grant and back to dashboard
+def test_add_new_grant_from_dashboard_and_back_to_dashboard(page: Page, domains: str):
+    dashboard = DashboardPage(page, domains)
+    dashboard.when_goto_dashboard()
+    dashboard.then_load_dashboard()
+    dashboard.then_click_add_a_new_grant()
+    create_grant = CreateGrantPage(page)
+    create_grant.then_load_create_grant_page()
+    create_grant.then_fill_non_welsh_competitive_grant_details()
```

Comment:
I think I would like to see `then_click_add_a_new_grant` return an instance of the create grant page? Since a user will end up on the create grant page after clicking 'add a new grant', it doesn't feel representative for the test to manually load the grant page.

I think we could end up with this?

```suggestion
    create_grant = dashboard.then_click_add_a_new_grant()
    create_grant.then_fill_non_welsh_competitive_grant_details()
```

And apply the same principle to any other actions which cause the user to change page. What do you think?

Reply:
We don't load the grant page we just pick the elements on that method let me rename that method which represent work it does